bin/plugin/commands/common.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+const glob = require( 'fast-glob' );
+const fs = require( 'fs' );
+
+/**
+ * @typedef WPModuleDescription
+ *
+ * @property {string} name        Module name.
+ * @property {string} description Module description.
+ */
+
+/**
+ * Returns a promise resolving to the module description list string for the `readme.txt` file.
+ *
+ * @param {string} modulesDir Modules directory.
+ *
+ * @return {Promise<[]WPModuleDescription>} Promise resolving to module description list.
+ */
+exports.getModuleDescriptions = async ( modulesDir ) => {
```

Comment:
This is now in here since pretty much the same code was needed to extract the module header translations (`npm run translations` command) as it was needed to update the list of modules within the readme (the new `npm run readme` command).